<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="coin10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>+10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coin50">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>+50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coin100">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>+100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coin500">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>+500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getExchange">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>exchange</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyCoffee">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>coffee (100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyTea">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>tea (150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyMilk">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>milk (200)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
